### DISCUSSION TITLE
Is it safe to ignore "status" field updates for all resources for reconciliation trigger?

### DISCUSSION THREAD

**Discussion by andrii-korotkov-verkada** [argoproj/argo-cd#18179] [Q&A]
Hello, I hope your day is going well.
I've analyzed the logs about what triggers reconciliation and found that ~60% of that was HorizontalPodAutoscaler and ScaledObject. Those update their status section quite frequently. I've found an example on just disabling reconciliation when status section is changed for any resource (see https://argo-cd.readthedocs.io/en/stable/operator-manual/reconcile/#system-level-configuration), but want to confirm whether it's safe to do so. Is this a good approach?

**Comment by ChristianCiach** [comment]
I just asked myself the same question. I am not completely sure, but I *think* this would probably render any[ custom health checks](https://argo-cd.readthedocs.io/en/stable/operator-manual/health/#custom-health-checks) that are based on the `status` attribute useless, because the health check would probably never trigger anymore.

I didn't try this. This is just my assumption.
